Assets/CutsceneFade.cs

```diff
@@ -0,0 +1,36 @@
+using UnityEngine;
+using UnityEngine.UI;
+using Yarn.Unity;
+
+public class CutsceneFade : MonoBehaviour
```

Comment:
nit: `[RequireComponent(typeof(Image))` since you call `GetComponent` and don't check for null